runtime/vm/swalk.c

```diff
@@ -256,7 +257,13 @@ UDATA  walkStackFrames(J9VMThread *currentThread, J9StackWalkState *walkState)
 	}
 #endif 
 
+	loop_count = 0;
 	while(1) {
+		loop_count = loop_count + 1;
+		if (loop_count > 4096) {
```

Comment:
For any one wondering where this seemingly magic number probably came from: OpenJDK uses `2 * MaximumStackHeight == 4096` as its loop breaker: https://github.com/openjdk/jdk/blob/master/src/hotspot/share/prims/forte.cpp#L393, https://github.com/openjdk/jdk/blame/bfbfdfa4289ac351182ed2a643b110ef68b2b60a/hotspot/src/share/vm/prims/forte.cpp#L327